apps/explorer/components/Contract/index.tsx

```diff
@@ -2,152 +2,162 @@
+import { humanBytes, humanSiacoin } from '@siafoundation/units'
 import { EntityList, EntityListItemProps } from '@siafoundation/design-system'
 import { DatumProps, ExplorerDatum } from '../ExplorerDatum'
 import { ContentLayout } from '../ContentLayout'
 import { ContractHeader } from './ContractHeader'
 import { siacoinToFiat } from '../../lib/currency'
 import { useExchangeRate } from '../../hooks/useExchangeRate'
+import {
+  ExplorerFileContract,
+  FileContractElement,
+  SiacoinOutput,
+} from '@siafoundation/explored-types'
 
 type Props = {
-  contract: SiaCentralContract
+  previousRevisions: FileContractElement[]
+  currentHeight: number
+  contract: ExplorerFileContract
   rates?: SiaCentralExchangeRates
-  renewedTo?: SiaCentralContract
-  renewedFrom?: SiaCentralContract
+  renewedTo?: ExplorerFileContract
+  renewedFrom?: ExplorerFileContract
 }
 
-export function Contract({ contract, rates, renewedFrom, renewedTo }: Props) {
+export function Contract({
+  previousRevisions,
+  currentHeight,
+  contract,
+  rates,
+  renewedFrom,
+  renewedTo,
+}: Props) {
   const exchange = useExchangeRate(rates)
   const values = useMemo(() => {
     return [
       {
         label: 'file size',
         copyable: false,
-        value: humanBytes(contract.file_size),
+        value: humanBytes(contract.fileContract.filesize),
       },
       {
         label: 'payout',
         copyable: false,
-        value: siacoinToFiat(contract.payout, exchange),
-        comment: humanSiacoin(contract.payout),
+        value: siacoinToFiat(contract.fileContract.payout, exchange),
+        comment: humanSiacoin(contract.fileContract.payout),
       },
       {
         label: 'transaction ID',
         entityType: 'transaction',
-        entityValue: contract.transaction_id,
+        entityValue: contract.confirmationTransactionID,
       },
       {
         label: 'merkle root',
-        value: contract.merkle_root,
+        value: contract.fileContract.fileMerkleRoot,
       },
       {
         label: 'unlock hash',
-        value: contract.unlock_hash,
+        value: contract.fileContract.unlockHash,
       },
       {
         label: 'proof confirmed',
         copyable: false,
-        value: String(contract.proof_confirmed),
-      },
-      {
-        label: 'negotiation height',
-        copyable: false,
-        value: contract.negotiation_height?.toLocaleString() || '-',
-      },
-      {
-        label: 'negotiation time',
-        copyable: false,
-        value:
-          contract.negotiation_timestamp !== '0001-01-01T00:00:00Z'
-            ? humanDate(contract.negotiation_timestamp, {
-                dateStyle: 'medium',
-                timeStyle: 'short',
-              })
-            : '-',
+        value: String(contract.valid),
       },
+      // {
+      //   label: 'negotiation height',
+      //   copyable: false,
+      //   value: contract.negotiation_height?.toLocaleString() || '-',
```

Comment:
Use confirmation index for these fields. Get the time with `blockHeightToTime` method.